vector/src/main/java/im/vector/app/features/roommemberprofile/RoomMemberProfileController.kt

```diff
@@ -200,30 +199,10 @@ class RoomMemberProfileController @Inject constructor(
 //                divider = !state.isMine,
 //                action = { callback?.onOverrideColorClicked() }
 //        )
-=======
-        if (!state.isMine) {
-            buildProfileAction(
-                    id = "direct",
-                    editable = false,
-                    title = stringProvider.getString(R.string.room_member_open_or_create_dm),
-                    action = { callback?.onOpenDmClicked() }
-            )
-        }
```

Comment:
@giomfo do you want to add this part ?

Reply:
we are supposed to have it already - we will discuss in direct with @Claire1817